docs/source/about/guides/linux/headless_ssh.rst

```diff
@@ -0,0 +1,498 @@
+       Identifier "Monitor0"
+       Option "Enable" "true"
+   EndSection
+
+   Section "Device"
+       Identifier "Card0"
+       Driver "nvidia"
+       VendorName "NVIDIA Corporation"
+       Option "MetaModes" "1920x1080"
+       Option "ConnectedMonitor" "DP-0"
+       Option "ModeValidation" "NoDFPNativeResolutionCheck,NoVirtualSizeCheck,NoMaxPClkCheck,NoHorizSyncCheck,NoVertRefreshCheck,NoWidthAlignmentCheck"
+   EndSection
+
+   Section "Screen"
+       Identifier "metaScreen"
+       Device "Card0"
+       Monitor "Monitor0"
+       DefaultDepth 24
+       Option "TwinView" "True"
+       SubSection "Display"
+           Modes "1920x1080"
+       EndSubSection
+   EndSection
+
+.. note::
+   The ``ConnectedMonitor`` tricks the GPU into thinking a monitor is connected,
+   even if there is none actually connected! This allows a virtual display to be created that is accelerated with
+   your GPU! The ``ModeValidation`` option disables valid resolution checks, so you can choose any
+   resolution on the host!
+
+   **References**
+
+   - `issue comment on virtual-display-linux
+     <https://github.com/dianariyanto/virtual-display-linux/issues/9#issuecomment-786389065>`__
+   - `Nvidia Documentation on Configuring TwinView
+     <https://download.nvidia.com/XFree86/Linux-x86/270.29/README/configtwinview.html>`__
+   - `Arch Wiki Nvidia#TwinView <https://wiki.archlinux.org/title/NVIDIA#TwinView>`__
+   - `Unix Stack Exchange - How to add virtual display monitor with Nvidia proprietary driver
+     <https://unix.stackexchange.com/questions/559918/how-to-add-virtual-monitor-with-nvidia-proprietary-driver>`__
+
+
+Uinput Permissions Workaround
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Steps**
+
+We can use ``chown`` to change the permissions from a script. Since this requires ``sudo``,
+we will need to update the sudo configuration to execute this without being prompted for a password.
+
+#. Create a ``sunshine-setup.sh`` script to update permissions on ``/dev/uinput``. Since we aren't logged into the host,
+   the udev rule doesn't apply.
+#. Update user sudo configuration ``/etc/sudoers.d/<user>`` to allow the ``sunshine-setup.sh``
+   script to be executed with ``sudo``.
+
+.. note::
+   After I setup the :ref:`udev rule <about/usage:linux>` to get access to ``/dev/uinput``,
+   I noticed when I sshed into the host without physical login, the ACL permissions on ``/dev/uinput`` were not changed.
+   So I asked `reddit
+   <https://www.reddit.com/r/linux_gaming/comments/14htuzv/does_sshing_into_host_trigger_udev_rule_on_the/>`__.
+   I discovered that SSH sessions are not the same as a physical login.
+   I suppose it's not possible for SSH to trigger a udev rule or create a physical login session.
+
+**Setup Script**
+
+This script will take care of any precondtions prior to starting up sunshine.
+Create a script named something like ``sunshine-setup.sh``:
+
+.. code-block:: bash
+
+   #!/bin/bash
+   chown $(id -un):$(id -gn) /dev/uinput
```

Comment:
This doesn't work, since this script is running as root, this is equivalet to `chown root:root /dev/uinput`